doc/release-notes/11519-my-data-api-bug-using-session-auth-for-spa.md

```diff
@@ -0,0 +1,5 @@
+## BUG Fix
+
+Call to API /api/mydata/retrieve was using JSF session data to get the user (if logged in on another browser tab). This caused the other user's data to be returned.
```

Comment:
```suggestion
The API call to ``/api/mydata/retrieve`` was previously using JSF session data to identify the user. This could result in another user's data being returned if they were logged in via JSF and making requests from a different client within the same domain, such as the SPA. This issue has been resolved: the API now prioritizes Bearer Token authentication, using session cookie authentication only as a fallback for JSF requests.
```